### PR TITLE
New date picker integration

### PR DESCRIPTION
**Pull request by rohit-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by prateek-giddh — APPROVED** [review]

File: apps/web-giddh/src/app/daybook/daybook.component.html [hunk 1]
```diff
@@ -23,7 +23,7 @@ <h1 class="section-title">DayBook</h1>
 
             <div class="form-inline d-block flex-row flex-nowrap">
                 <div class="form-group custom-datepicker" (click)="showGiddhDatepicker($event)">
-                    <i class="icon-calender calendar-addon"></i>
+                    <i><img src="assets/images/custom-calender.svg"></i>
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/daybook/daybook.component.html:26]: Why you removed above I class and make an image, I have do correct it

File: apps/web-giddh/src/app/expenses/expenses.component.html [hunk 2]
```diff
@@ -9,10 +9,9 @@
                         <div class="d-flex">
 
                             <div class="clearfix mr-1">
-                                <div class="input-group input-calender">
-                                    <span class="input-group-addon" id="basic-addon1"><i class="icon-calender"></i></span>
-                                    <input type="text" class="form-control" placeholder="Daterangepicker" daterangepicker
-                                        [options]="datePickerOptions" (applyDaterangepicker)="bsValueChange($event)">
+                                <div class="input-group input-calender custom-datepicker" (click)="showGiddhDatepicker($event)">
+                                    <i><img src="assets/images/custom-calender.svg"></i>
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/expenses/expenses.component.html:13]: Why you removed above I class and make an image

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/expenses/expenses.component.html:13]: add I tag

File: apps/web-giddh/src/app/invoice/eWayBill/eWayBill/eWayBill.component.html [hunk 3]
```diff
@@ -2,11 +2,9 @@
 <div class="clearfix pt-15 pb-15">
     <div class="col-xs-12 col-sm-12">
         <div class="d-inline-block pull-left mr-2 middle ewayBill-date">
-            <div class="input-group">
-                <span class="input-wrapper">
-                    <input type="text" name="daterangeInput" daterangepicker [options]="datePickerOptions"
-                        (applyDaterangepicker)="selectedDate($event)" class="form-control" />
-                </span>
+            <div class="custom-datepicker" (click)="showGiddhDatepicker($event)">
+                <i><img src="assets/images/custom-calender.svg"></i>
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/invoice/eWayBill/eWayBill/eWayBill.component.html:6]: Remove image tag and add I tag

File: apps/web-giddh/src/app/invoice/generate/invoice.generate.component.html [hunk 4]
```diff
@@ -26,11 +26,10 @@
...
                     <div class="form-group date-advance-search pending-advance-search" style="flex: 1 1 auto;">
                         <div class="input-group" id="dp-parent" style="margin-right:8px; margin-bottom:0;">
-                            <span class="input-wrapper">
-                                <input type="text" name="dateRangeGenerate" id="dateRangeGenerate" daterangepicker
-                                    [options]="datePickerOptions" (applyDaterangepicker)="bsValueChange($event)"
-                                    class="form-control" />
-                            </span>
+                            <div class=" input-group custom-datepicker" (click)="showGiddhDatepicker($event)">
+                                <i><img src="assets/images/custom-calender.svg"></i>
+                                <input type="text" name="selectedDateRange" [value]="(selectedDateRangeUi) ? selectedDateRangeUi : ''" class="giddh-datepicker date-range-picker" />
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/invoice/generate/invoice.generate.component.html:31]: Remove image tag and add I tag

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html [hunk 5]
```diff
@@ -6,11 +6,9 @@
             <form #invoiceForm="ngForm" (ngSubmit)="getVoucherByFilters(invoiceForm)" novalidate
                 class="form-inline input-grp-dateRangePicker pr">
                 <div class="form-group mrR1 d-flex" style="flex: 1 1 auto;">
-                    <div class="input-group">
-                        <span class="input-wrapper">
-                            <input type="text" name="dateRange" daterangepicker [options]="datePickerOptions"
-                                (applyDaterangepicker)="bsValueChange($event)" class="form-control daterange-picker" />
-                        </span>
+                    <div class=" input-group custom-datepicker" (click)="showGiddhDatepicker($event)">
+                        <i><img src="assets/images/custom-calender.svg"></i>
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html:10]: Remove image tag and add I tag

File: apps/web-giddh/src/app/invoice/preview/models/advanceSearch/invoiceAdvanceSearch.component.html [hunk 6]
```diff
@@ -227,9 +227,9 @@ <h3 class="modal-title bg" id="modal-title">Advance Search </h3>
 
                         <div>
 
-                            <div class="pr-0">
-                                <input type="text" name="invoiceDate" daterangepicker class="form-control w-100 cursor-pointer"  [options]="dateRangePickerOptions"
-                                    (applyDaterangepicker)="dateRangeChanged($event)" />
+                            <div class="custom-datepicker" (click)="showGiddhDatepicker($event)">
+                                <i><img src="assets/images/custom-calender.svg"></i>
+                                <input type="text" name="selectedDateRange" [value]="(selectedDateRangeUi) ? selectedDateRangeUi : ''" class="giddh-datepicker date-range-picker" />
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/invoice/preview/models/advanceSearch/invoiceAdvanceSearch.component.html:232]: Remove image tag and add I tag

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html [hunk 7]
```diff
@@ -6,12 +6,10 @@
...
             <div class="mrR1 d-flex w-100">
                 <div class="d-flex">
-                <div class="input-group">
-                    <span class="input-wrapper">
-                        <input type="text" name="dateRange" daterangepicker [options]="datePickerOptions"
-                            (applyDaterangepicker)="onDateChange($event)" class="form-control cursor-pointer" />
-                    </span>
-                </div>
+                    <div class="custom-datepicker" (click)="showGiddhDatepicker($event)">
+                        <i><img src="assets/images/custom-calender.svg"></i>
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:10]: Remove image tag and add I tag

File: apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html [hunk 8]
```diff
@@ -1,15 +1,10 @@
...
         <div class="mt-15 input-grp-dateRangePicker d-flex">
-            <div class="input-group">
-                <div class="form-group">
-                    <span class="icon-calender" (click)="dp.toggle()"></span>
-                    <input type="text" placeholder="Select Date" class="form-control" #dp="bsDaterangepicker"
-                        [(ngModel)]="datePicker" (bsValueChange)="changeFilterDate($event)" bsDaterangepicker
-                        [bsConfig]="bsConfig" container="body">
-                </div>
+            <div class="custom-datepicker" (click)="showGiddhDatepicker($event)">
+                <i><img src="assets/images/custom-calender.svg"></i>
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html:6]: Remove image tag and add I tag

File: apps/web-giddh/src/app/search/components/sidebar-components/search.sidebar.component.html [hunk 9]
```diff
@@ -11,9 +11,10 @@
                 </div>
                 <div class="form-group mr-15">
                     <label>From - To</label>
-                    <input type="text" name="daterangeInput" daterangepicker [options]="datePickerOptions"
-                        (hideDaterangepicker)="selectedDate($event)" (applyDaterangepicker)="selectedDate($event)"
-                        class="form-control" />
+                    <div class="custom-datepicker" (click)="showGiddhDatepicker($event)">
+                        <i><img src="assets/images/custom-calender.svg"></i>
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/search/components/sidebar-components/search.sidebar.component.html:15]: Remove image tag and add I tag

**Review by shalinee-walkover — APPROVED** [review]

**Pull request merged**